src/Squisher.js

```diff
@@ -114,13 +114,53 @@ class Squisher {
         }
     }
 
-    updateHelper(node, playerFrames, whitelist) {
+    updateHelper(node, playerFrames, whitelist, scale) {
+        if (this.game.getRoot() === node) {
+            scale = {
+                x: (100 - config.BEZEL_SIZE.x) / 100,
+                y: (100 - config.BEZEL_SIZE.y) / 100
+            }
+        }
+
         if (!this.ids.has(node.node.id)) {
             this.ids.add(node.node.id);
             node.addListener(this);
         }
 
-        const squished = squish(node.node);
+        let squished;
+        if (scale) {
+            const scaledNode = unsquish(squish(node.node));
```

Comment:
I feel like you explained to me why we need to squish then unsquish, but could you add a comment here explaining it, because I already don't remember the reasoning behind it, I just remember it was important